js/ui/appDisplay.js

```diff
@@ -1109,7 +1109,7 @@ var AppSearchProvider = class AppSearchProvider {
 
                 const app = this._appSys.lookup_app(appID);
```

Comment:
Might be simpler (but equivalent, so only rework it if you care) to instead do:
```
if (!app)
    return false;
```
above. Should slightly reduce the possibility of further bugs in this area.

Reply:
I didn't do that because these conditions are added in different patch sets, and I think it would make the next rebase harder to intertwingle these changes.